internal/arp/arp.go

```diff
@@ -94,15 +89,14 @@ func (a *Announce) Close() error {
 	return a.client.Close()
 }
 
-// SetBalancer implementes.. bla bla bla
-// Now only uses an net.IP.
+// SetBalancer implementes adds ip to the set of announced address.
 func (a *Announce) SetBalancer(name string, ip net.IP) {
 	a.Lock()
 	defer a.Unlock()
 	a.ips[name] = ip
 }
 
-// DeleteBalancer deletes...
+// DeleteBalancer an address from the set of address we should announce.
```

Comment:
Missing word: deletes. Github + timezones makes review RTT annoying, so I'll just fix it up in a followup commit